config/app.default.php

```diff
@@ -9,7 +9,7 @@
      * Development Mode:
      * true: Errors and warnings shown.
      */
-    'debug' => true,
+    'debug' => env('DEBUG', true),
```

Comment:
@tmatsuo I don't like fuzzy values.

IME it's not normal for a boolean env var to allow fuzzy-values, therefore I don't feel it should be allowed here.

> If you need to set '0' or '1' for debug, it's very confusing.

I think that's only confusing if you don't typically use env vars

> Likewise, if it needs to be exactly 'true' and 'false', it's also confusing.

Luckily nobody suggested that so it doesn't need considering =)

Whatever @josegonzalez says though.